apps/kol-amelamdim/pages/api/upload-file.ts

```diff
@@ -31,41 +35,22 @@ export default async function handler(req, res) {
         secretAccessKey: process.env.S3_SECRET_KEY,
       });
 
+      const fileLocation = `${selectedCategory}/${formData.files.sharedFile.originalFilename}`;
       const params = {
         Bucket: process.env.AWS_BUCKET_NAME,
-        Key: `parashat-shavoa/${data.files.sharedFile.originalFilename}`,
+        Key: fileLocation,
         Body: contents,
-        Metadata: {
-          something: 'wow!',
-        },
       };
 
-      // const response = await s3
-      //   .listObjectsV2({
-      //     Bucket: process.env.AWS_BUCKET_NAME,
-      //     Prefix: 'parashat-shavoa',
-      //   })
-      //   .promise();
-      //
-      // console.log(response);
-
-      // s3.getObject(
-      //   {
-      //     Bucket: process.env.AWS_BUCKET_NAME,
-      //     Key: data.files.sharedFile.originalFilename,
-      //   },
-      //   (err, data) => {
-      //     console.log(data);
-      //   }
-      // );
-
-      s3.upload(params, (err, data) => {
+      s3.upload(params, (err, _) => {
         if (err) {
-          res.status(400).json(API_ERRORS.uploadFileError);
+          res.status(400).send(API_ERRORS.uploadFileError);
```

Comment:
we can go with the convention of returning json, what do you say about it ?

Reply:
Fixed man